xcb/linux.cmake

```
@@ -0,0 +1,155 @@
+find_package(PkgConfig REQUIRED)
+
+pkg_check_modules(
+  XCB
+  REQUIRED
```

pkg_check_modules 还不够，需要link
```suggestion
pkg_check_modules( XCB  REQUIRED IMPORTED_TARGET
```

```
target_link_libraries(${PROJECT_NAME} PRIVATE
    PkgConfig::XCB
    )
```